.github/workflows/pylint.yml

```diff
@@ -1 +1,28 @@
+name: Pylint
 
+on: 
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.11
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ./live_pipeline/requirements.txt
+        pip install matplotlib
+        pip install seaborn
+        pip install streamlit
```

Comment:
Instead of individual libraries here I'd change to using requirements from the archive pipeline and dashboard. Should make this more maintainable if other libraries were added in the future.